siteimprove/admin/js/siteimprove.js

```diff
@@ -50,6 +55,33 @@
         ]);
         return;
       }
+      _si.push(['onHighlight', function(highlightInfo) {
+        // Remove highlight tag wrapper
+        $(".si-highlight").contents().unwrap();
+        // Create an span tag for every highlight
+        $.each(highlightInfo.highlights, function(index, highlight) {
+          var $element = $(highlight.selector);
+          var text = $element.text();
+
+          if (highlight.offset) {
+            var start = highlight.offset.start;
+            var length = highlight.offset.length;
+
+            var before = text.substr(0, start);
+            var highlighted = text.substr(start, length);
+            var after = text.substr(start + length);
+
+            $element.html(before + "<span class='si-highlight'>" + highlighted + "</span>" + after);
+          } else {
+            $element.html("<span class='si-highlight'>" + text + "</span>");
+          }
+          
+          //Scroll to the target element
+          $([document.documentElement, document.body]).animate({
+            scrollTop: $(".si-highlight").offset().top
+          }, 1500);
```

Comment:
Remember offset by wp-admin bar. 

I suggest using jQuery to first get the height of it, and then use it as the offset rather than using a static value.

Reply:
Done.